bucket/opentyrian2000.json

```diff
@@ -0,0 +1,42 @@
+{
+    "version": "2000.20240218",
+    "description": "Fork of OpenTyrian with support for Tyrian 2000",
+    "homepage": "https://github.com/KScl/opentyrian2000",
+    "license": "GPL-2.0-or-later",
+    "notes": [
+        "Place Tyrian 2000 game data files in:",
+        "",
+        "    $persist_dir\\data"
+    ],
```

Comment:
Note for a future PR: The data files are freeware as linked in the repository's README file, so Scoop could download them using multiple URLs support (`url` can be an array of URLs to download).